config/bootstrap.php

```diff
@@ -1,16 +1,25 @@
 <?php
 
-use Symfony\Component\Dotenv\Dotenv;
+use Symfony\Component\Yaml\Yaml;
 
 require dirname(__DIR__).'/vendor/autoload.php';
 
-/**
- * Removed the dotenv parsing from this file to support the legacy parameters.yml config to prevent refactoring the configuration in Deploy
- * The original source copuld be found here:
- * https://github.com/symfony/recipes/blob/master/symfony/framework-bundle/4.2/config/bootstrap.php
- */
+$parametersPath = dirname(__DIR__).'/config/openconext/parameters.yaml';
+$parameters = Yaml::parseFile($parametersPath);
+$parameters = $parameters['parameters'];
+$requiredParameters = ['app_env', 'app_debug', 'app_secret'];
 
-$_SERVER += $_ENV;
-$_SERVER['APP_ENV'] = $_ENV['APP_ENV'] = ($_SERVER['APP_ENV'] ?? $_ENV['APP_ENV'] ?? null) ?: 'dev';
-$_SERVER['APP_DEBUG'] = $_SERVER['APP_DEBUG'] ?? $_ENV['APP_DEBUG'] ?? 'prod' !== $_SERVER['APP_ENV'];
-$_SERVER['APP_DEBUG'] = $_ENV['APP_DEBUG'] = (int) $_SERVER['APP_DEBUG'] || filter_var($_SERVER['APP_DEBUG'], FILTER_VALIDATE_BOOLEAN) ? '1' : '0';
+// Test if required parameters are set
+if (0 !== count(array_diff($requiredParameters, array_keys($parameters)))) {
+    throw new RuntimeException(sprintf(
+        'Required parameters are not configured, required params are: %s, configure them in %s',
+        implode(', ', $requiredParameters),
+        $parametersPath
+    ));
+}
+
+$_SERVER['APP_ENV'] = $parameters['app_env'];
+$_SERVER['APP_DEBUG'] = $parameters['app_debug'];
+$_SERVER['APP_SECRET'] = $parameters['app_secret'];
+
+filter_var($_SERVER['APP_DEBUG'], FILTER_VALIDATE_BOOLEAN) ? '1' : '0';
```

Comment:
Please add the env-var override option for the APP_ENV as seen here: `https://github.com/OpenConext/Stepup-tiqr/blob/feature/rejuvination/config/bootstrap.php#L24C1-L27C2`